packages/sign_in_with_apple/sign_in_with_apple/example/lib/main.dart

```diff
@@ -63,7 +73,9 @@ class _MyAppState extends State<MyApp> {
                     if (credential.familyName != null)
                       'lastName': credential.familyName!,
                     'useBundleId':
-                        Platform.isIOS || Platform.isMacOS ? 'true' : 'false',
+                        !kIsWeb && (Platform.isIOS || Platform.isMacOS)
```

Comment:
The `Platform` enum is also not available on web?

Reply:
No, accession this crashes. There is a similar thing in `dart:html`, but that can only tell you if you're on the web.

Found using the compiler constant as a workaround to only access this when possible. (I guess another option would be to write 2 files with the same interface, for native and web, and then use another conditional import.)